Rock.Messaging.SQS/Configuration/SQSConfiguration.cs

```diff
@@ -8,13 +8,15 @@ public SQSConfiguration()
         {
             MaxMessages = 3;
             AutoAcknowledge = true;
+            ParallelHandling = false;
```

Comment:
Defaults to false, why set explicitly?

Reply:
My motivation around adding it explicitly was readability, but that's fair - I'll remove this.